src/contentTypeConfig.js

```diff
@@ -52,6 +52,12 @@ let contentTypeConfig = {
             '"primary_cta_text": select(dateTime(published_on) > dateTime(now()) && dateTime(enrollment_start_time) > dateTime(now()) => "Notify Me", "Start Challenge")',
             'challenge_state',
             'challenge_state_text',
+            'description',
```

Comment:
Description is an object field, so has a complex structure. We may update the format later, but for the moment are using the the raw html. 
```suggestion
            `"description": ${descriptionField}`,
```